search-app/src/cmr/search/services/query_walkers/temporal_range_extractor.clj

```diff
@@ -2,11 +2,12 @@
   "Defines protocols and functions to extract keywords from a query. The extraction is done
   during pre-processing of the query to get the temporal conditions result feature."
   (:require
-   [clojure.string :as str]
-   [cmr.common-app.services.search.query-model :as cqm]
-   [cmr.common.services.errors :as errors]
    [cmr.common.util :as util]
-   [cmr.search.models.query :as qm]))
+   ;;[cmr.search.models.query :as qm]
```

Comment:
remove comment if not using

Reply:
this was a test to see if the run time would complain about it, if it did, I would have moved it to the import section.